payments-app-extension-credit-card/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,24 @@
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+target = "credit_card"
+
+merchant_label = "Credit Credit Payments App Extension"
+start_payment_session_url = "https://api.paymentprovider.com/endpoint"
+start_refund_session_url = "https://api.paymentprovider.com/refund"
+start_capture_session_url = "https://api.paymentprovider.com/capture"
+start_void_session_url = "https://api.paymentprovider.com/void"
+start_verification_session_url = "https://api.paymentprovider.com/verification"
+supported_countries = ["US"]
+supported_payment_methods = ["visa"]
+supports_3ds = false
+supports_installments = false
+supports_deferred_payments = false
+test_mode_available = true
+api_version = "2021-07"
+encryption_certificate = {}
```

Comment:
What information does this hash include? It would be nice to provide an example of its format in this template, since this field will be required for credit card payment apps

Reply:
I was expecting a gid or uuid, since the partner simply selects an existing certificate within the dashboard 🤔 Could this be a security issue? Do we validate in the backend if the certificate in the configuration belongs to the partner?

Reply:
as far as i can tell there is a light form of validation (mostly on the presence of keys and that the certificate string starts with certain characters). the frontend in this case loads partner certificates and offers the user to pick the appropriate cert.